app/connectors/ApiConnector.scala

```diff
@@ -41,9 +41,9 @@ class ApiConnector @Inject() (httpClient: HttpClient, appConfig: AppConfig)(impl
     httpClient.GET[Departures](url)(implicitly, headers, ec)
   }
 
-  def getMRN(departureId: String)(implicit hc: HeaderCarrier): Future[Option[MovementReferenceNumber]] = {
+  def getMRN(departureId: String)(implicit hc: HeaderCarrier): Future[MovementReferenceNumber] = {
     val url = s"${appConfig.apiUrl}/movements/departures/$departureId"
-    httpClient.GET[Option[MovementReferenceNumber]](url)(implicitly, headers, ec)
+    httpClient.GET[MovementReferenceNumber](url)(HttpReads[MovementReferenceNumber], headers, ec)
```

Comment:
@BledarFeratiEY hey, I had a merge conflict here. I took my changes because. I had just tested them. It looks like we were both fixing the same issue, happy to take your changes. Maybe call to chat about it?